internal/app/focus.go

```diff
@@ -14,6 +14,12 @@ func (f *FocusManager) nextInChain(current fyne.Focusable) fyne.Focusable {
 	var first, next fyne.Focusable
 	found := current == nil // if we have no starting point then pretend we matched already
 	driver.WalkObjectTree(f.canvas.Content(), fyne.NewPos(0, 0), func(obj fyne.CanvasObject, _ fyne.Position) bool {
+		if w, ok := obj.(fyne.Disableable); ok {
```

Comment:
This (and below) could be shortened to "...; ok && w.Disabled() {" could it not? (and avoid the extra if)